code.user.js

```diff
@@ -2566,6 +2566,17 @@
             }
         }
 
+        function getPriceValueAsInt(listing) {
+            // Match number part from any currency format
+            const results = listing.match(/\d[0-9 .,]*(\.|,\d{0,2})?/gm);
```

Comment:
`/\d[0-9 .,]+/` I think this works just as well?

Reply:
Yep and looks more elegant